tests/buyers/views/test_buyers.py

```diff
@@ -2255,7 +2244,7 @@ def test_404_if_framework_is_not_live_or_expired(self):
     def test_get_responses(self):
         brief = mock.Mock()
 
-        with po(buyers, 'get_sorted_responses_for_brief') as m:
+        with mock.patch.object(buyers, 'get_sorted_responses_for_brief', autospec=True) as m:
             result = self.instance.get_responses(brief)
 
         assert result == m.return_value
```

Comment:
```

    def test_get_responses(self):
        brief = mock.Mock()

        with mock.patch.object(buyers, 'get_sorted_responses_for_brief', autospec=True) as m:
            result = self.instance.get_responses(brief)
        print result
        print m.return_value
        assert result == m.return_value

        m.assert_called_once_with(brief, self.instance.data_api_client)
```
```
(venv) ben@ben-desktop:~/gds/branches/digitalmarketplace/digitalmarketplace-buyer-frontend$ py.test -s tests/buyers/views/test_buyers.py::TestDownloadBriefResponsesView::test_get_responses
============================= test session starts ==============================
platform linux2 -- Python 2.7.12, pytest-2.9.1, py-1.4.33, pluggy-0.3.1
rootdir: /home/ben/gds/branches/digitalmarketplace/digitalmarketplace-buyer-frontend, inifile: setup.cfg
plugins: hypothesis-3.6.1, cov-2.2.0
collected 16 items 

tests/buyers/views/test_buyers.py <MagicMock name=u'get_sorted_responses_for_brief()' id='140126571028304'>
<MagicMock name=u'get_sorted_responses_for_brief()' id='140126571028304'>
.

=========================== 1 passed in 2.58 seconds ===========================
```

I don't think that was doing what it was assumed to be doing. Removing unless there are arguments otherwise.